### PR TITLE
update missing Chinese translation(CAPI and Deprecated features) for latest and v2.8

### PR DESCRIPTION
**Pull request by JacieChao**
## Description

Update missing Chinese translation for:
- Cluster API (CAPI)
- Deprecated features

### CODE REVIEW

**Comment by JacieChao** [comment]
@jianghang8421 Hi hang, would you please help to review the docs? Thank you.

File: i18n/zh/docusaurus-plugin-content-docs/version-2.8/integrations-in-rancher/cluster-api/cluster-api.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Cluster API (CAPI) 与 Rancher Turtles
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/zh/integrations-in-rancher/cluster-api"/>
+</head>
+
+[Rancher Turtles](https://turtles.docs.rancher.com/) 是一个 [Rancher 扩展](../rancher-extensions.md)，通过提供 Cluster API (CAPI) 和 Rancher 之间的集成来管理预置的 Kubernetes 集群的生命周期。使用 Rancher Turtles，你可以：
```
Comment (jianghang8421) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.8/integrations-in-rancher/cluster-api/cluster-api.md:9]: `预置`按照总结的术语表中翻译为`配置`更合适。

File: i18n/zh/docusaurus-plugin-content-docs/version-2.8/integrations-in-rancher/cluster-api/cluster-api.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+- 通过在 CAPI 预置的集群中安装 Rancher Cluster Agent，将 CAPI 集群导入 Rancher。
```
Comment (jianghang8421) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.8/integrations-in-rancher/cluster-api/cluster-api.md:11]: `预置`按照总结的术语表中翻译为`配置`更合适。

**Review by jianghang8421 — COMMENTED** [review]
Others LGTM

**Review by jianghang8421 — APPROVED** [review]
LGTM

**Pull request merged**